mithril-test-lab/mithril-end-to-end/benchmark-aggregator.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+set -e
+
+# Debug mode
+if [ -v DEBUG ]; then
+    set -x
+fi
+
+# Check if all env vars are set 
+if [ -z "${MIN_SIGNERS}" ]; then
+    MIN_SIGNERS=1
+fi
+
+if [ -z "${MAX_SIGNERS}" ]; then
+    echo Missing environment variable: MAX_SIGNERS
```

Comment:
Would you consider redirecting theses messages to StdErr?

Reply:
Done :heavy_check_mark: